pages/board/index.tsx

```diff
@@ -82,8 +84,10 @@ export default function Board() {
           <SelectBtn order={order} setOrder={setOrder} />
         </div>
         <div className={styles["bottom-wrap"]}>
-          {GeneralArticles.map((article) => (
-            <GeneralPost article={article} key={article.id} />
+          {generalArticles.map((article) => (
+            <Link href={`/${article.id}`}>
```

Comment:
이 부분도 동일하게 경로가 수정되어야 할 것 같아요.

```suggestion
            <Link href={`/board/${article.id}`}>
```